### PR TITLE
Add OSX CLI instructions

### PR DESCRIPTION
**Pull request by masklinn**
I'd have provided a pre-built workflow but they're annoying to distribute and inspect, so giving the instructions for people to create their own workflow is slightly less convenient for them but much more trustworthy.

Though I guess they could be provided as a "release" on this project or something? Just commit the entire bundle directory then zip it up for distribution? IDK.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -6,14 +6,42 @@ would put "unofficial" in the name.
...
+
+If your primary browser is Safari, you can also create a Service for this:
+
+* open Automator
+* choose New Document > Service
+* check "Output replaces selected text" in the header
+* in the right pane, search for "shell", drag "Run Shell Script" to the right 
+  pane
+* select "/bin/sh" as shell
+* replace the text (`cat`) by `sed 's/^/    /'
```
Comment (singron) [inline about.md]: Looks like you are missing a trailing `` ` ``?

**Review by singron — CHANGES_REQUESTED** [review]
Thanks for you contribution! I think I agree about distributing opaque files myself, especially since I don't use OSX and wouldn't be able to personally vouch for their integrity.

I spotted one small typo when I previewed and I'm happy to merge once it's fixed.

File: about.md (hunk 1 above)
Comment (masklinn) [reply about.md]: Should be fixed, thanks.

**Comment by masklinn** [comment]
> I think I agree about distributing opaque files myself

Yeah, they're not opaque per-se (it's a bundle with two XML plist files) but it still seems like as much work as re-doing it by hand. A more complex workflow might make sense to distribute directly but this is a pretty trivial one.

**Review by singron — APPROVED** [review]
Thanks again. I wasn't expecting contributions so quickly so this was a very pleasant outcome.

**Pull request merged**